instaloader/instaloader.py

```diff
@@ -1434,6 +1435,9 @@ def download_profiles(self, profiles: Set[Profile],
 
         .. versionchanged:: 4.8
            Add `latest_stamps` parameter.
+
+        .. versionchanged:: 4.12
+           Add `max_count` parameter.
```

Comment:
Would you please also describe the parameter in the :param: listing here in the method docstring?

Reply:
@aandergr